### PR TITLE
Introduce multi-lingual logout redirect

### PR DESCRIPTION
**Pull request by pablothedude**
The logout redirect utrl didn't work. This commit will make sure the
redirect uri will work and adds multilingual support.

### CODE REVIEW

**Comment by pablothedude** [comment]
https://www.pivotaltracker.com/story/show/161717981

File: src/Surfnet/StepupSelfService/SelfServiceBundle/Security/Authentication/Handler/LogoutSuccessHandler.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+     * @var string[]
+     */
+    private $logoutRedirectUrl;
+
+    /**
+     * @param TokenStorageInterface $tokenStorage
+     * @param string[] $locales
+     * @param string[] $logoutRedirectUrl
+     */
+    public function __construct(TokenStorageInterface $tokenStorage, array $locales, array $logoutRedirectUrl)
```
Comment (MKodde) [inline src/Surfnet/StepupSelfService/SelfServiceBundle/Security/Authentication/Handler/LogoutSuccessHandler.php]: Nice solution for this problem. Some open ends:
- The parameter `$locales` is never used. So either use them for verification and or picking the default language if the users preferred locale is not set.
- Consider adding some test coverage for this class.
- use Symfony\Component\Translation\TranslatorInterface; is not used. And can be removed from the 'imports' list.

File: src/Surfnet/StepupSelfService/SelfServiceBundle/Security/Authentication/Handler/LogoutSuccessHandler.php [hunk 2]
```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * Copyright 2016 SURFnet B.V.
```
[the remark this answers is not in the document]
Comment (pablothedude) [reply src/Surfnet/StepupSelfService/SelfServiceBundle/Security/Authentication/Handler/LogoutSuccessHandler.php]: Changed

File: src/Surfnet/StepupSelfService/SelfServiceBundle/Security/Authentication/Handler/LogoutSuccessHandler.php (hunk 1 above)
Comment (pablothedude) [reply src/Surfnet/StepupSelfService/SelfServiceBundle/Security/Authentication/Handler/LogoutSuccessHandler.php]: Removed the$locales parameter and removed the unused import. At the moment I can't justify the time for adding test coverage for the logoutsuccesshandler so I'll skip that.

**Pull request merged**